lib/runtime/wasmer/test_helpers.go

```diff
@@ -52,14 +52,25 @@ func setupConfig(t *testing.T, tt *trie.Trie, lvl log.Level, role common.Roles)
 		BaseDB:            runtime.NewInMemoryDB(t), // we're using a local storage here since this is a test runtime
 	}
 
+	var versions *testVersions
+	if targetRuntime == runtime.HOST_API_TEST_RUNTIME {
+		// Force state version to 0 since the host api test runtime
+		// does not implement the Core_version call so we cannot get the
+		// state version from it.
+		versions = &testVersions{
+			stateVersion: 0,
+		}
```

Comment:
When shall we bump this to 1 for the host api runtime? When we support the v1 state trie (finally) or would you see another milestone to bump this up for the host api test runtime? 🤔

Reply:
> i guess, do you want to test the imported trie functions w the different versions? if so you might need the two diff versions (or just override it)